lib/msal-common/src/telemetry/performance/PerformanceClient.ts

```diff
@@ -318,6 +318,7 @@ export abstract class PerformanceClient implements IPerformanceClient {
 
                     const completedEvent = this.endMeasurement(event);
                     if (completedEvent) {
+                        completedEvent.isIncomplete = true;
```

Comment:
I'd expect that an incomplete measurement represents either a bug, a timeout or crash somewhere.  Please add a datapoint to log whether we had incomplete measurements on a given call.  Just one datapoint for now, if we find there are tons of them we'll look at something more comprehensive if needed so we can debug/understand.

Reply:
Would a single data point with an array of manually completed events work? This should be both inexpensive to store and easy to debug.

Reply:
Very easy to debug, expensive to store in telemetry.  For telemetry a boolean just to see how prevalent it is, would be great. 
For tracing/logging, an array of manually completed events would be fine and that would help with one-off debugging in the near term.